js/comments.js

```diff
@@ -0,0 +1,61 @@
+import { renderPack } from './util.js';
+
+const COMMENTS_PACK_AMOUNT = 5;
+
+const commentsCount = document.querySelector('.social__comment-count');
+const commentsLoader = document.querySelector('.comments-loader');
+const commentsList = document.querySelector('.social__comments');
+const commentTemplate = commentsList.querySelector('.social__comment');
+
+let shownComments = 0;
+
+const createComment = ({avatar, message, name}) => {
+  const commentElement = commentTemplate.cloneNode(true);
+  const commentAvatar = commentElement.querySelector('.social__picture');
+  const commentText = commentElement.querySelector('.social__text');
+
+  commentAvatar.src = avatar;
+  commentAvatar.alt = name;
+  commentText.textContent = message;
+
+  return commentElement;
+};
+
+let onLoadMoreClick = null;
+
+const createSliceRenderer = (comments) => {
```

Comment:
Тоже самое что писал в прошлый раз
```suggestion
const createSliceRenderer = (comments) => () => {
```